src/scenes/FunderReport/FunderReportView.tsx

```diff
@@ -67,7 +66,7 @@ const FunderReportView = () => {
     return selectedReport?.startDate?.split('-')[0];
   }, [selectedReport]);
 
-  const reportName = selectedReport ? getReportName(selectedReport) : '';
+  const reportName = userFundingEntity?.projects?.[0].dealName || '';
```

Comment:
Does this need a question mark after `[0]` as well?
`userFundingEntity?.projects?.[0]?.dealName`

Also should this variable be renamed now from `reportName`?